lighthouse-core/config/config-helpers.js

```diff
@@ -183,13 +183,26 @@ function resolveModule(moduleIdentifier, configDir, category) {
     return require.resolve(cwdPath);
   } catch (e) {}
 
-  const errorString =
-    'Unable to locate ' +
-    (category ? `${category}: ` : '') +
-    `${moduleIdentifier} (tried to require() from '${__dirname}' and load from '${cwdPath}'`;
+  let errorString = 'Unable to locate ' + (category ? `${category}: ` : '') +
+    `\`${moduleIdentifier}\`.
+     Tried to require() from these locations:
+       ${__dirname}
+       ${cwdPath}`;
 
   if (!configDir) {
-    throw new Error(errorString + ')');
+    throw new Error(errorString);
+  }
+
+  if (moduleIdentifier.startsWith('lighthouse-plugin-')) {
+    // See if the module resolves in the context of the current working directory.
+    // Most useful to handle the case of Lighthouse plugin development.
+    const cwdPath = path.resolve(process.cwd(), '../', moduleIdentifier);
```

Comment:
yah that's a good idea. 

----

IMO the audit path that we expect in plugin.js doesn't make sense to me.  eg.

https://github.com/connorjclark/lighthouse-plugin-yt/blob/136869d31885b3894d12882350989c8c9112c4b6/plugin.js#L6

wdyt about for v6 we would change that so that the audit paths defined in plugin.js are relative

oh wait. i think that'd simplify plugin dev but it'd break plugin use?  seems like it would in theory.

@connorjclark @brendankenny does that sound right? that we essentially need the plugin-name in the path so the resolution for a user works?

Reply:
We can do this change w/o breaking plugin users. Just need to try one more path in our resolution strategy (prefix the name of the plugin to the audit module path, if the initial resolution didnt work)